app/controllers/api/v0/api_controller.rb

```diff
@@ -156,7 +156,11 @@ def omni_search
   end
 
   def delegates
-    paginate json: UserGroup.delegate_regions.flat_map(&:active_users)
+    delegate_users = UserGroup.roles_of_group_type(
+      UserGroup.group_types[:delegate_regions],
+      includes_params: [:group, { user: [:delegate_roles, :delegate_role_metadata, :active_roles] }],
+    ).flat_map(&:user)
```

Comment:
It used to be `active_users`, now you're (flat-)mapping to just `users`. Is this intended or a whoopsie?

Reply:
Kindly pinging this comment :)